memdown.js

```diff
@@ -116,6 +116,24 @@ MemIterator.prototype._test = function () {
   return true
 }
 
+MemIterator.prototype._seek = function (target) {
+  var options = this._options
+
+  var tree = this.db._store
+
+  if (this.keyAsBuffer && !Buffer.isBuffer(target)) {
+    target = Buffer.from(String(target))
+  }
+
+  if ([target].filter(ltgt.filter({ options })).length !== 1) { throw Error('key is out-or-range') }
```

Comment:
An out-of-range target should just end the iterator, like in `leveldown`

Reply:
should conform now and passes the tests with seek: true